packages/ui-contexts/src/AvatarUrlContext.ts

```diff
@@ -6,7 +6,7 @@ export type AvatarUrlContextValue = {
 	getUserPathAvatar: {
 		(username: string, etag?: string): string;
 		(params: { userId: string; etag?: string }): string;
-		(params: { username: string; etag?: string }): string;
+		(params: { username?: string; etag?: string }): string;
```

Comment:
how is this possible? a function that accepts an empty object will return something?

Reply:
@ggazzo did you checked the function?

Reply:
sure and still makes no sense rendering something if you have no info about thats why this params are wrong

you are trying to fix a different problem with a "naive" solution


your problem is the `/invite @something`  breaking right ?


for a small fraction the autocomplete uses one render type for a different item list thats why username is undefined there. but this is the real problema.

 even this change solves the issue, I think you are covering the wrong place